vector-im/element-android
# [Location sharing] - Make stop of a live from another device possible (PSF-1060)

## Type of change


## Content


Moving the sending of the stopping state event outside the Android service used to start and send location updates.
Observing the live status in DB from the same Android service to stop sending locations when live has ended and to stop the service if there is no more active lives.

## Motivation and context


Closes #6349

## Screenshots / GIFs





## Tests


2 devices are needed

- Ensure lab flag for live location sharing is enabled
- Start a live on a device
- Stop the live from another device (logged in with the same account)
- Check the live is correctly stopped and the location sharing service is closed in the first device

## Tested devices

- OS version(s): Android 11, 10

## Checklist
